### PR TITLE
Fix System.Transactions tests to fix CI

### PR DESCRIPTION
**Pull request by joperezr**
FYI: @weshaggard @stephentoub 

Fixing the System.Transaction tests build failure that is hitting our CIs.

### CODE REVIEW

File: src/System.Transactions/tests/System.Transactions.Tests.builds [hunk 1]
```diff
@@ -2,7 +2,9 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <Project Include="System.Transactions.Tests.csproj" />
+    <Project Include="System.Transactions.Tests.csproj">
+    	<TestTFMs>netcoreapp1.1</TestTFMs>
```
Comment (stephentoub) [inline src/System.Transactions/tests/System.Transactions.Tests.builds]: Nit: looks like there's a tab in here

**Comment by stephentoub** [comment]
Thanks! LGTM

File: src/System.Transactions/tests/System.Transactions.Tests.builds (hunk 1 above)
Comment (joperezr) [inline src/System.Transactions/tests/System.Transactions.Tests.builds]: You are right, fixed and rebased.

**Comment by joperezr** [comment]
@dotnet-bot test Innerloop Windows_NT Release Build and Test please

**Comment by weshaggard** [comment]
LGTM

**Pull request merged**